lib/alma/bib_item.rb

```diff
@@ -120,6 +120,10 @@ def missing_or_lost?
       !!process_type.match(/MISSING|LOST_LOAN/)
     end
 
+    def techserv_unassigned_or_intref?
```

Comment:
This method name seems too specific and perhaps not something that should be added to a general use application.  So I might consider moving this to helper and perhaps adding a higher order function that can take a predicate to apply as a filter (open to debate so use your judgement... just bringing up as a question).

However, is it possible that in the future whatever should make this true or false will change to be something other than that the holding location matches `/techserv|UASSIGNED|interf/` (i.e. because now we also want things that match `foo`)?

If so, then perhaps a more general method name like `unwanted_holding_location?`, will suffice.  Also a name like `unwanted_holding_location?` is easier for other developers who aren't familiar with the specific details of our need.

Of course, what is wanted/unwanted is very specific to the use case, thus my comment about using a helper method.  But a method that can be configured in some way could work here too.

Reply:
@dkinzer I was talking about with with @sensei100 about this issue before and and I agree about moving the specificity out of here and into cob. I thought `filter_by_holding_location(array_of_locations)` would work, as it would expose the functionality but allow the specifics locations to be defined in our codebase.

Reply:
@bibliotechy agreed, something like that would work really nicely here.

Reply:
@dkinzer Can you please clarify what you would like to happen here?  I'm unclear about whether I am supposed to be moving all of this code to tul_cob, or leaving more general parts here.  We already have a method that retrieves location codes, so maybe only a helper method in tub_cob is needed?